highlights.js

```diff
@@ -0,0 +1,174 @@
+import { shuffle } from './util.js';
+import { imageRecs, viewer } from './main.js';
+
+const animationDuration = 15000; // milliseconds
```

Comment:
Change this number to set how long the animation takes to go from one highlight to another.